docs/source/dashboard.rst

```diff
@@ -0,0 +1,130 @@
+Dashboard Diagnostics
+=====================
+
+Profiling parallel code can be challenging, but the :doc:`Dask distributed scheduler <scheduling>` 
+provides live feedback via its interactive dashboard. 
+
+## Include dashboard picture status page
+
+To be able to to use the dashboard diagnostics `Bokeh <https://docs.bokeh.org>`_ needs to 
+be installed. A link that redirects to the dashboard will prompt whenever the scheduler is 
+created, and you can access the link when you create a client.
+
+.. code-block:: python
+
+   from dask.distributed import Client
+   client = Client()  # start distributed scheduler locally. 
+   client            
+
+.. figure:: images/dashboard_link.png
```

Comment:
This screenshot doesn't look quite right. For me it looks like this.

<img width="1470" alt="image" src="https://user-images.githubusercontent.com/1610850/152148413-f3c8b922-9f7a-46fb-87b5-d24095575c41.png">

Reply:
Mmh, that looks much better. 
I've never been able to get the proper look on Firefox on my end, and I forgot how it actually looked. I can try a different browser and see if I can get a better capture.

Reply:
I think this is just a minor CSS difference between the classic Jupyter notebook and JupyterLab. I prefer the latter, of course :)